models/pytorch2onnx/ort_run_frozen.py

```diff
@@ -29,6 +34,10 @@ def get_numpy_dtype(onnx_dtype):
             return np.float32
```

Comment:
remove `float32` & `float64`? Float type in ONNX only includes `float16`, `float`, `double`. Ref: https://github.com/onnx/onnx/blob/63a586914d51fdfd6d8668e81de043e3d9b0b382/onnx/defs/data_type_utils.h#L28

Reply:
sure, done